src/daemon/daemon.cpp

```diff
@@ -373,14 +386,24 @@ std::vector<mp::NetworkInterface> validate_extra_interfaces(const mp::LaunchRequ
 
     if (!request->network_options().empty())
     {
+        std::string full_name =
+            (request->remote_name().empty() ? "release" : request->remote_name()) + ':' + request->image();
+
+        if (no_bridging_images.find(full_name) != no_bridging_images.end() &&
+            std::find_if(request->network_options().begin(), request->network_options().end(), [](const auto& net) {
+                return net.mode() == multipass::LaunchRequest_NetworkOptions_Mode_AUTO;
+            }) != request->network_options().end())
+        {
+            throw std::runtime_error(fmt::format("Automatic bridge configuration not implemented for {}", full_name));
```

Comment:
I wonder if we could guide people to `mode=manual` when reading this message. I would also replace "bridge" with "network" and "implemented" with "available". So, perhaps something like `Automatic network configuration not available for {}. Consider using manual mode.`. WDYT?

Reply:
Makes complete sense. Changed.